src/track/serato/tags.cpp

```diff
@@ -59,6 +59,50 @@ RgbColor SeratoTags::displayedToStoredTrackColor(RgbColor::optional_t color) {
     return RgbColor(colorCode);
 }
 
+double SeratoTags::findTimingOffsetMillis(const QString& filePath) {
+    // TODO: Find timing offset using mp3guessenc
```

Comment:
@ywwg pinging you again regarding the Serato Cue timing offsets. Since #2499 was merged, i rebased on master, so the diff should be much easier to read and review now.